package.json

```diff
@@ -27,9 +27,7 @@
     "icon": "globe",
     "loader": "ComponentLoader.coffee",
     "components": {
```

Comment:
Note that this list is no longer used since 0.7, so we could just remove it altogether